src/cml.js

```diff
@@ -281,7 +281,6 @@ class CML {
           log.job = id;
           log.status = 'job_ended';
           log.success = data.includes('Succeeded');
-          log.level = log.success ? 'info' : 'error';
```

Comment:
Reason removing this from the GitHub chunk on a BB pr?

Reply:
I resolve this following below